README.md

```diff
@@ -146,7 +146,7 @@ This takes you through the process of creating a file called a `package.json` wh
   "engines": {
     "node": ">= 0.10"
   },
-  "dependedncies": {
+  "dependencies": {
      "pre-commit": "^1.0.7"
```

Comment:
Following the conversation on Gitter, I'm going to assume this is resolved... In any case I guess it's something that we can quickly describe, and tell them not to worry about for the time being.